### ISSUE TITLE
Bot randomly disconnects from Discord.

### ISSUE THREAD

**Issue by deathspawn** [copygirl/cord#16]
I haven't seen any error in the backlog about this. Happens at random it seems. The bot will appear offline and stop relaying Discord messages to IRC. Oddly though, it still relays IRC messages to Discord and stays online on IRC.

**Comment by copygirl** [comment]
Yup, happens to me too and I haven't figured out why. The reason why IRC messages are still relayed to Discord is that even without a direct connection, messages can be sent using a REST endpoint or so.

**Comment by deathspawn** [comment]
I forgot I ran into a similar problem on another bot.

https://github.com/reactiflux/discord-irc/issues/159

Don't know if it'll help.

**Comment by copygirl** [comment]
I think this might have been resolved by updating the discord.js dependency at some point.
Haven't noticed our bot being gone from Discord like the issue describes in recent years.
